src/config/config_definition.cc

```diff
@@ -351,57 +351,58 @@ const std::string BoxKeys::playlistAllDirectories = "Playlist/allDirectories";
 const std::string BoxKeys::playlistRoot = "Playlist/playlistRoot";
 
 /// \brief default values for CFG_BOXLAYOUT_BOX
-static const std::vector<BoxLayout> boxLayoutDefaults {
+static const std::vector<BoxLayout> boxLayoutDefaults
```

Comment:
@KarlStraussberger , not sure why the linter job in github wants to keep this alignement. I'm using the clang-format supplied file and it always modifies this...

Not sure if about this warning
/usr/bin/docker run --name defafaf43d26b494b0444b8280ea1495c0582d_800514 --label defafa --workdir /github/workspace --rm -e "INPUT_SOURCE" -e "INPUT_EXCLUDE" -e "INPUT_EXTENSIONS" -e "INPUT_CLANGFORMATVERSION" -e "INPUT_STYLE" -e "INPUT_INPLACE" -e "HOME" -e "GITHUB_JOB" -e "GITHUB_REF" -e "GITHUB_SHA" -e "GITHUB_REPOSITORY" -e "GITHUB_REPOSITORY_OWNER" -e "GITHUB_REPOSITORY_OWNER_ID" -e "GITHUB_RUN_ID" -e "GITHUB_RUN_NUMBER" -e "GITHUB_RETENTION_DAYS" -e "GITHUB_RUN_ATTEMPT" -e "GITHUB_REPOSITORY_ID" -e "GITHUB_ACTOR_ID" -e "GITHUB_ACTOR" -e "GITHUB_TRIGGERING_ACTOR" -e "GITHUB_WORKFLOW" -e "GITHUB_HEAD_REF" -e "GITHUB_BASE_REF" -e "GITHUB_EVENT_NAME" -e "GITHUB_SERVER_URL" -e "GITHUB_API_URL" -e "GITHUB_GRAPHQL_URL" -e "GITHUB_REF_NAME" -e "GITHUB_REF_PROTECTED" -e "GITHUB_REF_TYPE" -e "GITHUB_WORKFLOW_REF" -e "GITHUB_WORKFLOW_SHA" -e "GITHUB_WORKSPACE" -e "GITHUB_ACTION" -e "GITHUB_EVENT_PATH" -e "GITHUB_ACTION_REPOSITORY" -e "GITHUB_ACTION_REF" -e "GITHUB_PATH" -e "GITHUB_ENV" -e "GITHUB_STEP_SUMMARY" -e "GITHUB_STATE" -e "GITHUB_OUTPUT" -e "RUNNER_OS" -e "RUNNER_ARCH" -e "RUNNER_NAME" -e "RUNNER_ENVIRONMENT" -e "RUNNER_TOOL_CACHE" -e "RUNNER_TEMP" -e "RUNNER_WORKSPACE" -e "ACTIONS_RUNTIME_URL" -e "ACTIONS_RUNTIME_TOKEN" -e "ACTIONS_CACHE_URL" -e "ACTIONS_RESULTS_URL" -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/gerbera/gerbera":"/github/workspace" defafa:f43d26b494b0444b8280ea1495c0582d  "--clang-format-executable" "/clang-format/clang-format13" "-r" "--color" "always" "--style" "file" "--inplace" "false" "--extensions" "h,cc" "--exclude" "./src/contrib" "./src"
/run-clang-format.py:246: SyntaxWarning: invalid escape sequence '\ '
  """